examples/canvas_glow/src/static_resource_loader.rs

```diff
@@ -0,0 +1,107 @@
+use pathfinder_gpu::resources::ResourceLoader;
+
+/// A hacky loader that includes the resources we need in the binary.
+///
+/// On desktop, we can use FilesystemResourceLoader (see canvas_minimal), but
+/// wasm32-unknown-unknown does not have a filesystem.
+///
+/// This does not have any platform dependencies, but will result in larger binaries, and slower
+/// compile times, so you may wish to implement an approach with different trade-offs.
```

Comment:
Hmm, have you considered using XHR or the Fetch API instead?

Reply:
Yes. This is likely something you would want to do outside of a minimal example. I'll explicitly mention this option in the comment. It may also be nice to provide such a loader in the future.

There are two reasons we may wish to avoid using a web API in this example. First, it's target-specific code that is not core to the example. Secondly, it requires a fair bit of setup, making the example harder to follow along with.

What do you think?

Reply:
Well, my main concern is that it hardcodes all the pathnames of resources, which adds some maintenance burden.